.circleci/config.yml

```diff
@@ -21,17 +21,19 @@ commands:
           command: echo 'export PATH="$(npm bin):$PATH"' >> $BASH_ENV
       - restore_cache:
           keys:
-            - v2-dependencies-{{ checksum "plasma_framework/package.json" }}
+            - v2-dependencies-{{ checksum "plasma_framework/package-lock.json" }}
       - run:
           name: Install dependencies
           working_directory: ~/repo/plasma_framework
           command: |
+            # FIXME: workaround for the error on old web3 version using git: https://circleci.com/gh/omisego/plasma-contracts/2973
+            # also the 'rm -rf node_modules/**/.git' workaround does not solve this
```

Comment:
I've tried to use `rm -rf node_modules/**/.git` which is stronger than the original one. However, it still have the problem in CI (though locally my Mac would be able to `npm install`).

Since it is only having issue in that specific package, I `rm` that single package instead. Also, as sb pointed out, rm the `.git` file is probably not a good practice: https://github.com/OpenZeppelin/openzeppelin-sdk/issues/929#issuecomment-499271999

Reply:
> Also, as sb pointed out, rm all .git file is probably not a good practice
The practice is acceptable, though, as we are running it in a container and there we never have some local copies of our repos that cannot be restored.

Reply:
ah, yeah I agree. But anyway that workaround is not working now so do we still want to keep that?

Reply:
How about this: let me think a little about the issue - shouldn't I find a solution till Monday, then we will remove the problematic dependency from the cache, therefore we will be able to remove the workaround (as we will replace it with another :joy:).